miner/worker.go

```diff
@@ -700,14 +710,20 @@ func (w *worker) commitTransaction(env *environment, tx *txpool.Transaction, rec
 		env.gasPool.SetGas(gp)
 		return nil, err
 	}
+
 	env.txs = append(env.txs, tx.Tx)
 	env.receipts = append(env.receipts, receipt)
 
+	gasUsed := new(big.Int).SetUint64(receipt.GasUsed)
+	env.profit.Add(env.profit, gasUsed.Mul(gasUsed, tx.Tx.GasPrice()))
+	env.blockReward.Add(env.blockReward, gasUsed.Mul(gasUsed, tx.Tx.GasPrice()))
```

Comment:
I guess no need, because the validator only query the balance of system account and compare, we just query the same balance at the end?

Reply:
We need record this to determine the `GasFee` in a Bid

Reply:
yeah, `GasFee` in a bid can be equal to the the balance of system account? Can it be simple compare to this?

Reply:
Removed

Reply:
The `blockReward` is still a field of `environment`, please remove it.

Reply:
Done